CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Changes
 
+## edge-21.9.4
+
+This edge is a release candidate for `stable-2.11.0`! It introduces a new
+`linkerd viz auth` command which shows metrics for server authorizations broken
+down by server for a given resource. It also shows the rate of unauthorized
+requests to each server.  This is helpful for seeing a breakdown of which
+authorizations are being used and what proportion of traffic is being rejected.
+
+It also fixes an issue in the proxy where  HTTP load balancers could continue
+trying to establish connections to endpoints that were removed from service
+discovery. In addition it improves the proxy's error handling so that it can
+signal to an inbound proxy when its peers outbound connections should be torn
+down.
+
+* Changed destination watch updates from `info` to `debug` to reduce the amount
+  of logs (thanks @bartpeeters!)
+* Added the `linkerd viz auth` command which shows metrics for server
+  authorizations broken down by server for a given resource
+* Fixed an issue where the policy controller's validating admission webhook
+  attempted to validate ServerAuthorizations when it should only be validating
+  Servers
+* Removed the dependency on `jest-enzyme` which is no longer maintained or used
+  by Linkerd
```

Comment:
This is totally an internal non-user facing item that can be removed :wink:

Reply:
```suggestion
```